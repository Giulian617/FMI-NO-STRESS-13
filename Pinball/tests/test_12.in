18 1000
16 15 9
14 16 1
8 14 9
9 8 5
7 9 10
4 7 4
10 4 7
11 10 4
3 11 9
2 3 4
5 2 5
18 5 6
1 18 7
13 1 10
6 13 9
12 6 10
17 12 10
