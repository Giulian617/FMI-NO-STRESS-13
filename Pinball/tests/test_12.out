470588239
529411769
